### DISCUSSION TITLE
EJBCA CE 7.9.0.2 on Ubuntu

### DISCUSSION THREAD

**Discussion by paolobrandazza** [Keyfactor/ejbca-ce#115] [Q&A] [answered]
Hi,
I'm new to EJBCA and I'm going to install EJBCA CE 7.9.0.2 on Ubuntu.
Given the fact I'm going to prepare a new Virtual Machine for this purpose I'm wondering if there is a preferred Ubuntu version as far as it concerns EJBCA support or can I start with the most recent Ubuntu LTS release ?

Thanks !

**Comment by primetomas** [comment, marked as answer]
The latest LTS is a great choice.
